### PR TITLE
Fix for #844 

### PR DESCRIPTION
**Pull request by pzas03**

### CODE REVIEW

**Comment by amikryukov** [comment]
:+1:

File: chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/aggregator/workload/DurationLogProcessor.java [hunk 1]
```diff
@@ -255,12 +248,14 @@ public StatisticsGenerator generate() throws IOException {
             persistAggregatedMetricValue(Math.rint(globalStatisticsCalc.getStandardDeviation()) / 1000D, latencyStdDevDesc);
 
             Long startTime = aggregationInfo.getMinTime();
-            Long endTime = aggregationInfo.getMaxTime();
```
Comment (dlatnikov) [inline chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/aggregator/workload/DurationLogProcessor.java]: can it be an issue of com.griddynamics.jagger.storage.fs.logging.ChronologyLogAggregator?

you are fixing it here, but looks like aggregation info max time is set incorrectly in the ChronologyLogAggregator

**Comment by Nmishin** [comment]
ok to test

File: chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/aggregator/workload/DurationLogProcessor.java [hunk 2]
```diff
@@ -254,13 +243,17 @@ public StatisticsGenerator generate() throws IOException {
             persistAggregatedMetricValue(Math.rint(globalStatisticsCalc.getMean()) / 1000D, latencyDesc);
             persistAggregatedMetricValue(Math.rint(globalStatisticsCalc.getStandardDeviation()) / 1000D, latencyStdDevDesc);
 
-            Long startTime = aggregationInfo.getMinTime();
-            Long endTime = aggregationInfo.getMaxTime();
+            Long startTime = aggregationInfo.getStartTime();
+            Long endTime = aggregationInfo.getEndTime();
+
+            log.info("qwe startTime: [{}], endTime: [{}]", startTime, endTime);
```
Comment (dlatnikov) [inline chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/aggregator/workload/DurationLogProcessor.java]: what is qwe? please make logs more informative. why do we need it on the info level?

File: chassis/core/src/main/java/com/griddynamics/jagger/storage/fs/logging/ChronologyLogAggregator.java [hunk 3]
```diff
@@ -20,19 +20,14 @@
...
 
-import com.google.common.io.Closeables;
-
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Iterator;
-import java.util.PriorityQueue;
-import java.util.Set;
+import java.util.*;
```
Comment (dlatnikov) [inline chassis/core/src/main/java/com/griddynamics/jagger/storage/fs/logging/ChronologyLogAggregator.java]: please, add setting in Idea to import all classes by name. No * imports allowed

File: chassis/core/src/main/java/com/griddynamics/jagger/storage/fs/logging/DurationLogEntry.java [hunk 4]
```diff
@@ -37,6 +37,11 @@ public void setDuration(long duration) {
         this.duration = duration;
     }
 
+    @Override
```
Comment (dlatnikov) [inline chassis/core/src/main/java/com/griddynamics/jagger/storage/fs/logging/DurationLogEntry.java]: please, add comment why is it necessary

File: chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/aggregator/workload/DurationLogProcessor.java (hunk 2 above)
Comment (pzas03) [reply chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/aggregator/workload/DurationLogProcessor.java]: fixed

File: chassis/core/src/main/java/com/griddynamics/jagger/storage/fs/logging/ChronologyLogAggregator.java (hunk 3 above)
Comment (pzas03) [reply chassis/core/src/main/java/com/griddynamics/jagger/storage/fs/logging/ChronologyLogAggregator.java]: fixed

File: chassis/core/src/main/java/com/griddynamics/jagger/storage/fs/logging/DurationLogEntry.java (hunk 4 above)
Comment (pzas03) [reply chassis/core/src/main/java/com/griddynamics/jagger/storage/fs/logging/DurationLogEntry.java]: done

**Pull request merged**